android/package.json

```diff
@@ -28,10 +28,10 @@
 	"compileSDKVersion": "29",
 	"vendorDependencies": {
 		"android sdk": ">=23.x <=29.x",
-		"android build tools": ">=28.0.3 <=29.x",
+		"android build tools": ">=29.0.2 <=29.x",
```

Comment:
feels like a bit of an odd version range to use here...

Reply:
To clarify, Android gradle tool v4.0.0 requires SDK build tool v29.0.2 or higher.
https://developer.android.com/studio/releases/gradle-plugin?buildsystem=ndk-build#4-0-0

There isn't a release version of SDK build v30.0.0 yet. It's currently an RC version.
https://developer.android.com/studio/releases/build-tools#notes

Also, we don't use these build tools directly anymore. Our Titanium SDK build scripts do pretty much everything via gradle (except for `adb` which is okay). But I'm trying to keep up with the version ranges here just in case something else depends on them that I'm not aware of.

Reply:
Or would you prefer we "uncap" the build tools version and not set a max?
I think that might be okay.